### PR TITLE
Fix issue with click indicator

### PR DESCRIPTION
**Pull request by ylvaselling**
Fix issue with click indicator making scroll content move when at bottom of the menu. This PR uses a different way of showing the click animation div (it is a child of the document now, not the node it refers to)

### CODE REVIEW

File: src/components/GettingStartedTour/TourPopup.jsx [hunk 1]
```diff
@@ -173,30 +184,30 @@ function Goal({ startSubscriptions, setIsFulfilled, hasGoals, stopSubscriptions,
...
-
-    return () => {
-      try {
-        tutorial.current[lastKey]?.removeChild(animationDiv.current);
-      }
-      catch(e) {
-        console.error("Error: " + e);
+  // If animation should be displayed
+  // Update the animation click so it is placed where it should be
+  if(animationDiv.current) {
```
Comment (WeirdRubberDuck) [inline src/components/GettingStartedTour/TourPopup.jsx:189]: nitpicky ,but please add a space before the first bracket in all the `if`s

Maybe we should start looking at setting up those linting rules?

**Review by WeirdRubberDuck — CHANGES_REQUESTED** [review]
Code looks good to me :) And seems to work well, but did find some other problems related to the indicator: 

-  indicator can get out of view, if one expands some menus over the item that is supposed to be indicated. But that was definitely a problem before as well and should be made a separate issue (also not sure if it's worth fixing). This was probably a problem before as well 

More serious: 
- The indicator can point to the wrong thing. Happened to me in the "change the color of the trail of mars" step. See video below
- Closing the tutorial does not make the indicator disappear! And a new one is added when the getting started tour is opened again


https://user-images.githubusercontent.com/8808894/196116076-1ea9b231-8fd8-44c8-8a60-0761da8afa9d.mov

File: src/components/GettingStartedTour/TourPopup.jsx (hunk 1 above)
Comment (ylvaselling) [reply src/components/GettingStartedTour/TourPopup.jsx]: Sounds good!

**Comment by ylvaselling** [comment]
The problem that the indicator points to the wrong thing and gets out of view are old problems, yes. Can make an issue for that the indicator points to the wrong item. That it gets out of view could probably wait until we get more user test feedback.

Fixed the issue that the indicator doesn't disappear when you close the popup.

**Comment by Roxeena** [comment]
I am not sure if these comments belong here since they relate more to the getting started tour in general, so feel free to move them to where you think they fit best. I managed to get my dev GUI working again and tested this out for the first time now and it looks great! I also had a few thoughts about it.

1. In the first task the goal is to zoom to a specific altitude and for me, the getting started tour window was placed just in front of the altitude part of the text in the top left. This meant that I had to move it to be able to read the altitude text, and a novice user might not know that this is possible and then have a hard time reading it. 

2. The phrasing of when we are going to go to the Moon is a little strange, it says that we can go to other **planets** too, and then we suggest going to the Moon. The Moon is not a planet so maybe rephrase this to other destinations or similar? Thinking that the ISS is not a celestial body so something more generic would be nice.

3. In the task of changing the speed of time, I had a hard time telling what the indicator was pointing at. Is it the quick adjustments or the fast forward/backward buttons? Maybe make this clearer somehow, or is this intentional that there are several ways to change the time?

5. In the task to choose a layer, it is mentioned that you can either choose the Utah or Sweden versions depending on where in the world you are, but the task is bound to the Utah version. Maybe let the user decide which one of them they want to enable and accept either of them as a valid solution. 

But in general, I think it looks great and works well 👍

**Comment by ylvaselling** [comment]
I started to fix things in another branch and realized I needed the changes in this branch, so I made a new PR that features more enhancements of the getting started tour. Closing this one
